### DISCUSSION TITLE
ISSUES WITH GITPOD ENVIRONMENT

### DISCUSSION THREAD

**Discussion by Kinglucky04** [Cyfrin/foundry-full-course-cu#2717] [General]
I couldn't successfully download and install the WSL on my windows 10, so i decided to use gitpod before i figure out a way to go about it.
but on creating the gitpod working space, my working space looks different from the tutorial. what am i getting wrong please

![gitpod shot](https://github.com/user-attachments/assets/05983e31-f852-4353-9618-a84081001bfb)

**Comment by EngrPips** [comment]
Things might have changed from when the course was made. The question is, does stuff work as expected?

**Reply by Kinglucky04** [reply]
Nope
The environment is entirely different and doesn't look like vacation.

I wish there us a way to upload the photo here

**Reply by EngrPips** [reply]
I have never used GitPod before, so I have little to no experience with it, but I trust someone who knows about it will see this post and help.

**Reply by Kinglucky04** [reply]
Alright thanks

**Reply by EngrPips** [reply]
Anytime.

**Comment by Kinglucky04** [comment]
Please you can reach out to me on X @king_lucky004

**Reply by EngrPips** [reply]
No worries, I will do when I go on X. I hardly go there lol.

**Comment by Kinglucky04** [comment]
Alright 
Due to this setback I have been unable to code for a few days now and I feel like I am seriously lagging behind

**Reply by EngrPips** [reply]
Why not look up some GitPod tutorials on YouTube and choose a recent tutorial that will most likely have the same GitPod as yours and you can learn how to work with the GitPod from the tutorial and then get back to coding.

**Reply by Kinglucky04** [reply]
Good idea
Thanks

**Reply by EngrPips** [reply]
Anytime, Friend. Let me know how that works out.

**Comment by Kinglucky04** [comment]
Okay
Will do
